### ISSUE TITLE
Uncaught TypeError: Cannot read property 'remove' of undefined

### ISSUE THREAD

**Issue by dodedodo** [leijou/php-server#16]
[Enter steps to reproduce:]

1. Start php server via atom
2. Stop php server via atom

**Atom**: 1.14.4 x64
**Electron**: 1.3.13
**OS**: Microsoft Windows 10 Pro
**Thrown From**: [php-server](https://github.com/leijou/php-server) package 0.7.0

### Stack Trace

Uncaught TypeError: Cannot read property 'remove' of undefined

```
At C:\Users\***\.atom\packages\php-server\node_modules\atom-message-panel\lib\MessagePanelView.js:189

TypeError: Cannot read property 'remove' of undefined
    at PhpServerView.MessagePanelView.remove (/packages/php-server/node_modules/atom-message-panel/lib/MessagePanelView.js:189:30)
    at PhpServerView.detach (/packages/php-server/node_modules/jquery/dist/jquery.js:5357:15)
    at Object.stop (/packages/php-server/lib/php-server.coffee:153:12)
    at /packages/php-server/lib/php-server.coffee:50:64)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:259:29)
    at /app.asar/src/command-registry.js:3:59
    at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/app.asar/node_modules/command-palette/lib/command-palette-view.js:210:32)
    at CommandPaletteView.module.exports.SelectListView.confirmSelection (/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
    at /app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:259:29)
    at /app.asar/src/command-registry.js:3:59
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:599:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:390:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/app.asar/src/window-event-handler.js:106:36)
    at /app.asar/src/window-event-handler.js:3:59)
```

### Commands

```
     -3:08.2.0 command-palette:toggle (input.hidden-input)
     -3:06.3.0 core:confirm (input.hidden-input)
     -3:06.3.0 php-server:start (input.hidden-input)
     -0:11.9.0 command-palette:toggle (input.hidden-input)
     -0:10.1.0 core:confirm (input.hidden-input)
     -0:10.1.0 php-server:stop (input.hidden-input)
```

### Non-Core Packages

```
git-control 0.8.2 
git-plus 7.3.3 
php-server 0.7.0 
```

**Comment by janstieler** [comment]
1+ have the Problem also.

**Comment by kloun1592** [comment]
1+ have the Problem also.

**Comment by Bkurq** [comment]
1+ Same here

**Comment by seanidzenga** [comment]
1+ ditto

**Comment by kgrossjo** [comment]
+1 I have the problem, as well.

**Comment by koheleth** [comment]
1+ have the problem also

**Comment by zisheng-ai** [comment]
+1 have the problem also

**Comment by leijou** [comment]
This was due to changes in how message panels attach in newer versions and has been fixed in v0.8.0
